### ISSUE TITLE
Linkit not working anymore in statamic v3

### ISSUE THREAD

**Issue by Oleafeon** [riasvdv/statamic-linkit#26]
Hi i have upgraded my site from statamic v2 to v3 and installed the plugin again but now it does not seem to work anymore

I am able to change the stuf in the blueprints but when i look at the page the button does not work anymore see screenshot below

![image](https://user-images.githubusercontent.com/20043831/90763572-36a00180-e2e7-11ea-83ea-f934c4d9957c.png)

this is what i have in my fieldset

```
-
  handle: buttons
  field:
      type: grid
      mode: stacked
      add_row: 'Een nieuwe knop toevoegen'
      fields:
          -
            handle: button
            field:
              type: link_it
              display: Knop
              types:
                - page
                - asset
                - url
              containers:
                  - main
                text: true
                newWindow: true
      display: Knoppen
```

**Comment by riasvdv** [comment]
```
              containers:
                  - main
                text: true
                newWindow: true
```
The indentation here seems wrong, is this because of pasting it in Github? It should look like this:

```
              containers:
                - main
              text: true
              newWindow: true
```

Could you also check your browser's console for errors?

**Comment by Oleafeon** [comment]
Thats because of github

Also the browsers does not give any error

**Comment by Oleafeon** [comment]
i also saw that the way the types where build different but after changing that it still did not work

This is the new fieldset i have
`-
                  handle: buttons
                  field:
                    type: grid
                    mode: stacked
                    add_row: 'Een nieuwe knop toevoegen'
                    fields:
                      -
                        handle: button
                        field:
                          type: link_it
                          display: Knop
                          types:
                            - asset
                            - url
                            - entry
                          containers:
                            - main
                          text: true
                          newWindow: true
                          listable: hidden
                          aria: false
                          append: false
                          collections:
                            - pages
                    display: Knoppen
                    listable: hidden
                    reorderable: true`

Wasnot be able to get the right indent in github so here is a screenshot

![image](https://user-images.githubusercontent.com/20043831/90765495-508f1380-e2ea-11ea-8d15-a03ca12b4e3e.png)

**Comment by Oleafeon** [comment]
@riasvdv after searching i found an error 
![image](https://user-images.githubusercontent.com/20043831/90770068-871c5c80-e2f1-11ea-891e-0d8e4796b5c1.png)

**Comment by riasvdv** [comment]
That looks to be a Bard issue, does everything else work inside the Bard field? Only LinkIt that is having issues?

**Comment by Oleafeon** [comment]
Everything else works fine

**Comment by ReneWeCode** [comment]
Confirming.
This is in a global, but not in a replicator, grid or anything.
And I don't get any errors.

![Skærmbillede 2020-08-20 kl  14 31 51](https://user-images.githubusercontent.com/43215436/90770370-fd20c380-e2f1-11ea-8a76-c69d08ab897b.png)

**Comment by Oleafeon** [comment]
in a grid field i dont get the error but it does not work aswel

Maybe its worth mentioning that everything is inside a replicator

**Comment by Oleafeon** [comment]
@riasvdv do you know how i can fix this?

**Comment by riasvdv** [comment]
Sorry about this, the JS build was doing something wrong, fixed in v2.1.1

**Comment by Oleafeon** [comment]
allright thank you

**Comment by Oleafeon** [comment]
@riasvdv sorry for opening again but i found out that the same problem happens when you only have one type in your blueprint

Like this
`-
   handle: button1
   field:
     types:
       - url
     newWindow: false
     text: true
     aria: false
     append: false
     display: 'knop 1'
     type: link_it
     icon: external-link
     listable: true`

**Comment by riasvdv** [comment]
Can you post your blueprint config? Easier to replicate then. I'll take a look tonight

**Comment by Oleafeon** [comment]
yes just updated it i dont know why its on one line

**Comment by riasvdv** [comment]
Should be fixed in v2.1.4!
